pkg/compiler/visitor.go

```diff
@@ -736,15 +736,39 @@ func (v *visitor) doVisitForExpressionStatement(ctx *fql.ForExpressionStatementC
 }
 
 func (v *visitor) doVisitMemberExpression(ctx *fql.MemberExpressionContext, scope *scope) (core.Expression, error) {
-	varName := ctx.Identifier().GetText()
+	var source core.Expression
+	var children []antlr.Tree
 
-	_, err := scope.GetVariable(varName)
+	identifier := ctx.Identifier()
 
-	if err != nil {
-		return nil, err
+	if identifier != nil {
+		varName := ctx.Identifier().GetText()
```

Comment:
Why not just `identifier.GetText()`?

Reply:
🤷‍♂️for clarity, I guess 😄

Reply:
Because I use the variable to do an existence check